src/App/App.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Container } from '@mui/material';
+
+const App = () => {
+    return (
+        <Container>
+        </Container>
+    )
```

Comment:
If the component has no child nodes, so it's better stay the component with self-closing tag: `<Container />`